hack/codegen.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -eux
+
+IS_CONTAINER=${IS_CONTAINER:-false}
+ARTIFACTS=${ARTIFACTS:-/tmp}
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
+
+if [ "${IS_CONTAINER}" != "false" ]; then
+  eval "$(go env)"
+  cd "${GOPATH}"/src/github.com/metal3-io/cluster-api-provider-metal3
+  export XDG_CACHE_HOME=/tmp/.cache
+  INPUT_FILES="api/v1alpha2/zz_generated.*.go api/v1alpha3/zz_generated.*.go api/v1alpha4/zz_generated.*.go baremetal/mocks/zz_generated.*.go"
+  cksum "$INPUT_FILES" > "$ARTIFACTS/lint.cksums.before"
```

Comment:
putting quotes around $INPUT_FILES make it behave as a single string. you need to remove the quotes and put a shellcheck exception before that

Reply:
Done!